libs/containers/Interaction.lua

```diff
@@ -136,7 +136,14 @@ function Interaction:_sendMessage(payload, files, deferred)
   if data then
     self._initialRes = true
     self._deferred = deferred or false
-    return true
+
+    if payload.flags and bit.band(payload.flags, messageFlag.ephemeral) ~= 0 then
+      -- message is ephemeral, therefore return true to indicate success because we can't reference the reply anyways
```

Comment:
Is it actually true that an ephemeral message won't have an object returned?

Reply:
It seems like it. The original comments a bit further down seemed to suggest that it was a problem with Discord not returning useful data when an ephemeral message was sent. I unfortunately couldn't find what I was looking for regarding this in the documentation. However, in practice when I was dealing with these messages I became quite frustrated with the fact that there was an inconsistency in returns so I decided to implement this change so that a message object is always returned. Not only would this just make the returns more consistent but also make it a bit more friendly for newcomers who, quite rightfully, likely expect it to return a message too.